docs/courses/using_morphometric_data.rst

```diff
@@ -0,0 +1,199 @@
+
+F. Zooming in
+~~~~~~~~~~~~~
+
+To zoom in for a closer look, first make sure that the Import3D tool's Zoom button is on (if it isn't, just click on it).
+Then click on the canvas, just to the right of the area of interest, and hold the mouse button down while dragging the cursor to the right. If it becomes necessary to re-center the image, click on the Translate button, then click on the canvas and drag the image into postion. To start zooming again, click on the Zoom button.
+
+Repeat as needed until you get what you want.
+
+.. image:: img/zoomp.gif
+    :align: center
+
+The irregular shape at the center, with the transverse orange lines, is the soma of a neuron. The green line is its principal axis, as identified by the Import3D tool. At least 9 neurites converge on it, and a fine red line connects the proximal end of each branch to the center of the soma.
+
+If you zoom in on the other green line and orange blob, you'll find another soma there.
+
+So by zooming in, it is possible to discover that this particular morphometric data file contained measurements from at least two different cells.
+
+To zoom out, make sure the Zoom button is on,
+then click near the right edge of the canvas and drag toward the left. To fit the image to the window, just use the graph's "View = plot" menu item
+
+A "litmus test" for models with complex architecture
+----------------------------------------------------
+
+Some morphometric reconstructions contain orphan branches, or measurement points with diameters that are (incorrectly) excessively small or even zero. Here's a test that can quickly detect such problems:
+
+1.
+    Use the data to create a model cell.
+
+2.
+    Insert the ``pas`` mechanism into all sections.
+
+    If you're dealing with a very extensive cell (especially if the axon is included), you might want to cut Ra to 10 ohm cm and reduce g_pas to 1e-5 mho/cm^2.
+
+3.
+    Turn on Continuous Export (if you haven't already).
+
+4.
+    Bring up a Shape Plot.
+
+5.
+    Turn this into a Shape Plot of Vm (R click in the Shape Plot and scroll down the menu to "Shape Plot". Release the mouse button and a color scale calibrated in mV should appear).
+
+6.
+    Examine the response of the cell to a 3 nA current step lasting 5 ms applied at the soma.
+
+    For very extensive cells, especially if you have reduced g_pas, you may want to increase both Tstop and the duration of the injected current to 1000 ms and use variable dt.
+
+Here's an example that used a toy cell:
+
+.. image:: img/v5.gif
+    :align: center
+
+Left: Vm at t = 0 ms. Right: Vm at 5 ms.
+
+Quantitative tests of anatomy
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This Python code checks for pt3d diameters smaller than 0.1 μm, and reports where they are found:
+
+.. code::
+    python
+
+    for sec in h.allsec():
+    for i in range(sec.n3d()):
+        if sec.diam3d(i) < 0.1:
+            print('%s %d %g' % (sec, i, sec.diam3d(i)))
```

Comment:
This is correct as-is, but let's change it to:

```python
print(f'{sec} {i} {sec.diam3d(i)}')
```

which is a more modern way of saying the same thing. (Keep the old indentation.)